modules/secrets-store-azure.adoc

```diff
@@ -0,0 +1,278 @@
++
+[source,terminal]
+----
+$ oc apply -f azure-provider.yaml
+----
+
+. Create a secret provider class to define your secrets store provider:
+
+.. Create a YAML file that defines the `SecretProviderClass` object:
++
+.Example `secret-provider-class-azure.yaml`
+[source,yaml]
+----
+apiVersion: secrets-store.csi.x-k8s.io/v1
+kind: SecretProviderClass
+metadata:
+  name: my-azure-provider                 <1>
+  namespace: my-namespace                 <2>
+spec:
+  provider: azure                         <3>
+  parameters:                             <4>
+    usePodIdentity: "false"
+    useVMManagedIdentity: "false"
+    userAssignedIdentityID: ""
+    keyvaultName: "kvname"
+    objects: |
+      array:
+        - |
+          objectName: tls-cert
+          objectType: secret
+    tenantId: "tid"
+----
+<1> Specify the name for the provider secret provider class.
+<2> Specify the namespace for the provider secret provider class.
+<3> Specify the provider as `azure`.
+<4> Specify the provider-specific configuration parameters.
+
+.. Create the `SecretProviderClass` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f secret-provider-class-azure.yaml
+----
+
+. Create a deployment to use this secret provider class:
+
+.. Create a YAML file that defines the `Deployment` object:
++
+.Example `deployment.yaml`
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-azure-deployment                            <1>
+  namespace: my-namespace                              <2>
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: my-storage
+  template:
+    metadata:
+      labels:
+        app: my-storage
+    spec:
+      containers:
+      - name: busybox
+        image: k8s.gcr.io/e2e-test-images/busybox:1.29
+        command:
+          - "/bin/sleep"
+          - "10000"
+        volumeMounts:
+        - name: secrets-store-inline
+          mountPath: "/mnt/key-store"
+          readOnly: true
+      volumes:
+        - name: secrets-store-inline
+          csi:
+            driver: secrets-store.csi.k8s.io
+            readOnly: true
+            volumeAttributes:
+              secretProviderClass: "my-azure-provider" <3>
+            nodePublishSecretRef:
+              name: secrets-store-creds                <4>
+----
+<1> Specify the name for the deployment.
+<2> Specify the namespace deployment.
+// TODO: Does this have to be the same ns as the secret provider class?
+<3> Specify the name of the secret provider class.
+<4> TODO
+// TODO: Need info about this field. The AWS test case doesn't use it
+
+.. Create the `Deployment` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f deployment.yaml
+----
+
+.Verification
+
+* Verify that you can access the secrets from AWS Secrets Manager in the pod volume mount:
+
+.. List the secrets in the pod mount:
++
+[source,terminal]
+----
+$ oc exec busybox-<hash> -n my-namespace -- ls /mnt/key-store/
```

Comment:
oc exec busybox-<hash> -n my-namespace -- ls /mnt/secrets-store/

Reply:
Updated to use /mnt/secrets-store throughout